### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,11 +15,121 @@ const goodsFromServer: string[] = [
...
+  };
+
+  reset = () => {
+    this.setState({ sortBy: '', isReversed: false });
+  };
+
+  render() {
+    const {
+      goods, isVisible, isReversed, sortBy,
+    } = this.state;
```
Comment (ihor-jpeg) [inline src/App.tsx:58]:
```suggestion
    const {
      goods,
      isVisible,
      isReversed,
      sortBy,
    } = this.state;
```

File: src/components/ListOfGoods/ListOfGoods.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+type Props = {
+  goods: string[],
+};
+
+export const ListOfGoods: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul className="list">
+      {
+        goods.map(good => (
+          <li key={good} className="list__item">{good}</li>
```
Comment (ihor-jpeg) [inline src/components/ListOfGoods/ListOfGoods.tsx:12]: map's callback also accepts the index. you can use that as a stronger version of the key.

**Review by ihor-jpeg — APPROVED** [review]
Well done!

File: src/components/ListOfGoods/ListOfGoods.tsx (hunk 2 above)
Comment (dariia-romanova) [reply src/components/ListOfGoods/ListOfGoods.tsx:12]: I thought index can't be used as a key. I get an error when I use it.

Comment (ihor-jpeg) [reply src/components/ListOfGoods/ListOfGoods.tsx:12]: Yes, you are right. I thought of an id for some reason. But the items in the array are just strings, so they don't have IDs. Never mind the previous comment.

**Pull request closed without merging**